src/Core/src/Platform/iOS/ViewExtensions.cs

```diff
@@ -63,13 +62,28 @@ public static void UpdateBackground(this UIView nativeView, IView view)
 			if (paint.IsNullOrEmpty())
 				return;
 
-			var backgroundLayer = paint?.ToCALayer(nativeView.Bounds);
+			if (paint is SolidPaint solidPaint)
+			{
+				Color backgroundColor = solidPaint.Color;
 
-			if (backgroundLayer != null)
+				if (backgroundColor == null)
+					nativeView.BackgroundColor = ColorExtensions.BackgroundColor;
+				else
+					nativeView.BackgroundColor = backgroundColor.ToNative();
+
+				return;
+			}
+
+			if (paint is GradientPaint gradientPaint)
```

Comment:
Maybe this should be an else type thing instead just in case we add new types. I am dreaming of that image brush world.

Reply:
Yes, I also want that ImageBrush!. Apply the changes.